modules/special-rules/pages/pulsar-x.adoc

```diff
@@ -0,0 +1,19 @@
+= Pulsar (X)
+
+This weapon is unique to the Eldar race, and it demonstrates their mastery of laser weaponry.
+
+The weapon shoots in rapid but powerful bursts, guided by advanced targeting technologies.
+As a result, the weapon can destroy multiple targets in a single salvo.
+
+---
+
+When you shoot with a {pulsar-x}:
+
+. Roll the dice listed in brackets, for example (D3+1) or (D6).
+. Resolve that many {macro-weapon} shots against the target detachment.
+
+IMPORTANT: Remember that a {pulsar-x} is a single super-heavy weapon.
```

Comment:
To avoid the weirdness of "a _Pulsar (X)_" (and to reinforce the main point slightly while I'm at it), perhaps something like this

```suggestion
IMPORTANT: Remember that {pulsar-x} always represents a single super-heavy weapon, regardless of the number of shots.
```

Reply:
Ooh, That's a bit of a tough one. We wouldn't say 'Remember that Death Ray always represents...', so I think we should use 'Remember that a {pulsar}...'.

So this?

```suggestion
IMPORTANT: Remember that a {pulsar} always represents a single super-heavy weapon, regardless of the number of shots.
```

Reply:
P.S. I did a minimal edit as the existing text had already been checked.

Reply:
> Ooh, That's a bit of a tough one. We wouldn't say 'Remember that Death Ray always represents...', so I think we should use 'Remember that a {pulsar}...'.

Haha, I wouldn't say that, but for a different reason. It depends on whether it is prose usage or 'keyword' usage, for want to a better description. (I often have to make this distinction in software documentation, and then adjust the text and formatting markup accordingly.)

- Prose usage: "Remember that a Death Ray weapon always represents..." — requires 'a' and 'weapon' and lacks special formatting for the term.
- Keyword usage: "Remember that _Death Ray_ always represents..." — omits 'a' and 'weapon' and uses the special formatting for the keyword.

I'll think about it some more. The hybrid usage of {pulsar} for both prose and keyword usage is already bugging me 😂 and maybe there's a neater way out of it.

Reply:
> P.S. I did a minimal edit as the existing text had already been checked.

Yeah, that's fine and is generally a good approach. I just often find that when I have new cause to edit something, then I'll spot and take other small opportunities to improve the text while I'm at it (or sometimes a new tweak creates new potential for ambiguity, so I seek to mitigate that).

Reply:
My suggestion at https://github.com/GameBrains/er-core/pull/43/files#r1186735306 will overcome part of this issue.

Then, for this note, I think we can follow through with something like this:

```suggestion
IMPORTANT: Although a {pulsar-x} shot results in a salvo of {macro-weapon} shots, it still only counts as a single super-heavy weapon.
```

Reply:
Scratch that — to tie in with my suggestion at https://github.com/GameBrains/er-core/pull/43/files#r1186738056, we can delete this line, and then modify the one after it.
```suggestion
```